app/javascript/src/ActiveDocs/customize/wrappedComponents/Parameters.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react'
+import 'ActiveDocs/customize/customizeOAS.scss'
+
+const AUTOCOMPLETE_PARAMS_MAP = {
+  'app_key': 'app_keys',
+  'app_id': 'app_ids',
+  'application_id': 'application_ids',
+  'user_key': 'user_keys',
+  'user_id': 'user_ids',
+  'account_id': 'account_ids',
+  'metric_name': 'metric_names',
+  'metric_id': 'metric_ids',
+  'backend_api_metric_name': 'backend_api_metric_names',
+  'service_id': 'service_ids',
+  'admin_id': 'admin_ids',
+  'service_plan_id': 'service_plan_ids',
+  'application_plan_id': 'application_plan_ids',
+  'account_plan_id': 'account_plan_ids',
+  'client_id': 'client_ids'
+  // TODO: secrets and tokens shouldn't autocomplete ?
+  // 'client_secret': 'client_secrets',
+  // 'service_token': 'service_tokens'
+  // 'access_token': 'access_token',
+}
+
+const getCustomParamValue = (customParamName, accountData) => {
+  const isAutocompletable = customParamName in AUTOCOMPLETE_PARAMS_MAP
+  if (!isAutocompletable) {
+    return undefined
+  }
+  const paramValue = accountData[AUTOCOMPLETE_PARAMS_MAP[customParamName]]
+    .map(item => {
+      return {name: item.name, value: item.value}
+    })
+  return paramValue.length > 0 ? paramValue : undefined
+}
+
+const AutocompleteList = ({autocomopleteValues, isOpen, customParamName, onClickAutocomplete}) => {
+  return (
+    <tr className={`autocomplete-row-${customParamName}`}>
```

Comment:
This table row is just to replicate current markup of swagger-ui.
We'll need some design here @cntlsn